backend/src/lib/controllers/settings-ctrl.ts

```diff
@@ -26,7 +26,7 @@ async function updateSettings(req: Request, res: Response) {
   }
 
   let { updatedAt } = req.body;
-  const { publishingGuidance, dateTimeFormat } = req.body;
+  const { publishingGuidance, dateTimeFormat, navbarTitle } = req.body;
```

Comment:
Can you please also add a test in the `settings-ctrl.test.ts` for this scenario of updating the navbarTitle.